pkgs/applications/networking/remote/freerdp/default.nix

```diff
@@ -14,13 +14,13 @@
 
 stdenv.mkDerivation rec {
   name = "freerdp-${version}";
-  version = "2.0.0-rc1";
+  version = "2.0.0-rc2";
 
   src = fetchFromGitHub {
     owner  = "FreeRDP";
     repo   = "FreeRDP";
     rev    = version;
-    sha256 = "0m28n3mq3ax0j6j3ai4pnlx3shg2ap0md0bxlqkhfv6civ9r11nn";
+    sha256 = "0330f5bxbsg3g3bxvmi1kcf9cs28i091l3ql46aqmb7xjdw3yhav";
```

Comment:
I get 

```
fixed-output derivation produced path '/nix/store/4200i7hxjb7m8hqpvj6l8vlk2bm9znx9-source' with sha256 hash '01cm9g4xqihnnc5d2w1zs8gabkv59p7fyjwi1cwpzv6s198xwbfs' instead of the expected hash '0330f5bxbsg3g3bxvmi
1kcf9cs28i091l3ql46aqmb7xjdw3yhav'
cannot build derivation '/nix/store/6dlyxp9ggswfkl006scl399xfcqnzv8c-freerdp-2.0.0-rc2.drv': 1 dependencies couldn't be built
```

Reply:
This is super strange - I've just tried running ```nix-prefetch-url -A freerdp.src``` and it produces the expected hash ```0330f5bxbsg3g3bxvmi1kcf9cs28i091l3ql46aqmb7xjdw3yhav```. Are you behind a misbehaving proxy?

Reply:
When happens if you run ```nix-prefetch-url -A freerdp.src``` ?

Reply:
after checkout out your PR I get this output:
```
$ nix-prefetch-url -A freerdp.src
unpacking...
path is '/nix/store/4200i7hxjb7m8hqpvj6l8vlk2bm9znx9-source'
01cm9g4xqihnnc5d2w1zs8gabkv59p7fyjwi1cwpzv6s198xwbfs
```
Is it possible that freerdp changed their tag?

Reply:
I'm an idiot. I had an overlay for freerdp because rc1 is broken. Merged in da4695009c85281ddf3903e73f40168a121d47b4 and will backport too.